udata_front/theme/gouvfr/datagouv-components/src/components/DatasetCard/DatasetCard.stories.ts

```diff
@@ -0,0 +1,93 @@
+import type { Meta, StoryObj } from '@storybook/vue3';
+import { DatasetCard } from ".";
+import type { DatasetV2 } from '../..';
+
+const meta = {
+  title: 'Components/DatasetCard',
+  component: DatasetCard,
+  tags: ['autodocs'],
+  parameters: {
+    docs: {
+      description: {
+        component: "The DatasetCard displays datas about a dataset and allows you to go to its page as well as its organization."
+      }
+    }
+  },
+} satisfies Meta<typeof DatasetCard>;
+
+export default meta;
+
+const args = {
+  dataset: {
```

Comment:
Can you move the dataset to its own variable and type it ? This should be easier to read